### DISCUSSION TITLE
Url spoof

### DISCUSSION THREAD

**Discussion by Darkhaxxor005** [Alygnt/Clifty#22] [Ideas]
There is tool called evil urls you can sse the url redirects of various popular websites and make the link look more safe

**Comment by Alygnt** [comment]
Thanks you for your suggestion. I will check it out and if possible I will implement it in next update.

**Reply by lovelysexly7** [reply]
and let the link work outside the browser

**Reply by Alygnt** [reply]
It works, But you have to select ngrok or cloudflare instead of localhost

**Reply by lovelysexly7** [reply]
how can i priven url from browser detecting not secured url

**Reply by lovelysexly7** [reply]
and is there away we can chat up here have question to asking you if you don't mind

**Reply by Alygnt** [reply]
Well, To prevent browser from detecting its a not secured url, You can send the link to victim without yourself testing it, in detail the first member who clicks link doesn't get that not secured page but the people who click the link 2nd or more they will get not secured page. And also make sure to send the http link not https
